monai/data/dataset.py

```diff
@@ -130,6 +130,9 @@ class PersistentDataset(Dataset):
     Subsequent uses of a dataset directly read pre-processed results from `cache_dir`
     followed by applying the random dependant parts of transform processing.
 
+    If want to update the input data and recompute cache content during training, just call `update_data()`
```

Comment:
Might be easier to say "During training call `update_data()` to update input data and recompute cache content.

Reply:
sure, updated in the latest commit.

Thanks.